internal/usecases/cli/cli.go

```diff
@@ -125,10 +125,10 @@ func (au *UseCases) validateTextOutputFilePath(cfg *config.Config) error {
 
 func (au *UseCases) validateFilePathAndExtension(cfg *config.Config, extension string) error {
 	if filepath.Ext(cfg.JSONOutputFilePath) != extension {
-		return fmt.Errorf("%snot valid file of type %s", messages.MsgErrorJSONOutputFilePathNotValid, extension)
+		return fmt.Errorf("%s %s", messages.MsgErrorJSONOutputFilePathNotValidExtension, extension)
 	}
 	if output, err := filepath.Abs(cfg.JSONOutputFilePath); err != nil || output == "" {
-		return errors.New(messages.MsgErrorJSONOutputFilePathNotValid + err.Error())
+		return fmt.Errorf("%s %e", messages.MsgErrorJSONOutputFilePathNotValidUnknown, err)
```

Comment:
This format %e is correct here? According to [fmt](https://pkg.go.dev/fmt) package the %e is fore scientific notation, e.g. -1.234456e+78. I think that we should use %w to [wrap](https://pkg.go.dev/errors) the error

Reply:
done